commitizen/cz/jira/jira.py

```diff
@@ -1,12 +1,20 @@
 import os
 
+from commitizen.config.base_config import BaseConfig
 from commitizen.cz.base import BaseCommitizen
 from commitizen.defaults import Questions
 
 __all__ = ["JiraSmartCz"]
 
 
 class JiraSmartCz(BaseCommitizen):
+    def __init__(self, config: BaseConfig):
+        super().__init__(config)
+        self.bump_map = None
```

Comment:
I think the values here is the ideal default values in `BaseCommitizen` and we should move the `defaults` to `ConventionalCommitsCz`